src/query/insert.rs

```diff
@@ -3,6 +3,16 @@ use crate::{
     QueryStatementBuilder, SelectExpr, SelectStatement, SimpleExpr,
 };
 
+/// Represents a value source that can be used in an insert query.
+///
+/// [`InsertValueSource`] is a node in the expression tree and can represent a raw value set
+/// ('VALUES') or a select query.
+#[derive(Debug, Clone)]
+pub enum InsertValueSource {
+    VALUES(Vec<Vec<SimpleExpr>>),
+    QUERY(SelectStatement),
+}
```

Comment:
I think it's better to `Box` the `SelectStatement`, i.e. `Box<SelectStatement>`.
CC @tyt2y3

Reply:
Done